src/components/signUpForm/SignUpForm.js

```diff
@@ -85,12 +86,7 @@ const SignUpForm = () => {
           />
         </div>
         <div className="form_input">
-          <input
-            type="submit"
-            className="form_input__field form_input__field--color"
-            value="sign up"
-            disabled={!validForm}
-          />
+          <Button disabled={!validForm} submit value="sign up" />
```

Comment:
I'm maybe arriving later to this but anyway I would like to know it :). Why are you using an input tag like your main button component? set submit prop to a button tag will give you the same behavior and is more customizable. 
https://html.com/attributes/button-type/

Reply:
Nice. There's no reason to use an `input` instead of `button`. I'll change it.